app/assets/javascripts/discourse/app/components/user-menu/notification-item.hbs

```diff
@@ -0,0 +1,32 @@
+<li class={{this.className}}>
+  <a
+    href={{this.url}}
+    title={{if this.linkTitle this.linkTitle}}
+    {{on "click" this.onClick}}
+  >
+    {{d-icon this.icon}}
+    <div>
+      {{#if this.label}}
+        {{#if this.wrapLabel}}
+          <span class={{if this.labelWrapperClasses this.labelWrapperClasses}}>
```

Comment:
Should work just fine:

```suggestion
          <span class={{this.labelWrapperClasses}}>
```

Reply:
This is so weird... I did what you suggest initially but then I noticed that it'd render the `class` (or `href`, `data-*` attrs etc.) with a blank value when the value was falesy instead of omitting the attribute altogether. However, I've tested it again now and it behaves as I'd expected (omits attribute entirely when the value is falsey). Is this because we're now on a newer version of Ember?

Reply:
possibly yes